fms/utils/generation.py

```diff
@@ -23,14 +23,15 @@ def _make_cache_contiguous(past_key_value_states):
 def generate(
     model: Union[Callable, torch.nn.Module],
     input_ids: torch.Tensor,
-    max_seq_len: int = 2048,
+    max_seq_len: int = 4096,
```

Comment:
any reason we're changing the default here? not that it's wrong or anything

Reply:
I just think all our models support at least 4096. 2048 seemed arbitrarily restrictive without a good reason (and makes it harder to have long context windows in chat). if we do have a model smaller than 4096 I'd expect it to be 1024 not 2048